docs/reference/openspec.md

```diff
@@ -7,16 +7,16 @@ localization_priority: Priority
 
 # API open specifications
 
-The Office JavaScript API open specifications provide information about new JavaScript APIs that are being designed for Excel, Word, and other host applications. You can review the open specifications in [branches](https://github.com/OfficeDev/office-js-docs/branches/all) of the **OfficeDev/office-js-docs** GitHub repository to learn about APIs that are planned for future releases and can provide feedback about the planned API features and design by logging issues in the GitHub repository.
+The Office JavaScript API open specifications provide information about new JavaScript APIs that are being designed for Excel, Word, and other host applications.
```

Comment:
Can we link to the What's new page where someone can get the latest info on the Preview Req Set? https://docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets